Merge download log+running lists

ByteHamster opened the pull request:
The `Downloads`»`Running` screen is mostly empty. My idea with this PR is to integrate the download log screen and the running downloads screen. This allows for reducing the number of horizontal tabs. I think a screen that is empty 99% of the time is rather useless. Feedback welcome, of course.

<img width="250" src="https://user-images.githubusercontent.com/5811634/115990486-cf4bf700-a5c3-11eb-8c9a-275ed18cb37f.png" />

keunes commented:
Big improvement :)
Just two comments:
* updated feeds aren't kept in the list; they disappear after succeeding (not the case in the current screen)
![Screenshot_20210427-234304.png](https://user-images.githubusercontent.com/11229646/116316886-e0a72600-a7b2-11eb-8339-d8d426f7dd90.png)
* If one episode is downloading and the other downloaded, the downloaded one 'time ago' counter keeps updating every second. Minor issue, but feels a bit odd. When the new download is finished, the 'time ago' counter stops

ByteHamster commented:
> updated feeds aren't kept in the list; they disappear after succeeding (not the case in the current screen)

That's also the case in the current version. This PR only changes displaying, not creating the logs. We only create a successful feed refresh log entry when the previous refresh failed. Otherwise, the list would be flooded with feed refresh items whenever we look for new episodes.

> If one episode is downloading and the other downloaded, the downloaded one 'time ago' counter keeps updating every second.

Not sure how to easily fix this. We update the whole list every time the running downloads change. Otherwise we would have to figure out what exactly changed (downloaders added, removed, updated). Would it look better if we just changed the text from "x seconds ago" to "just now"? While it does not make the problem go away, it becomes much less visible.

keunes commented:
> We only create a successful feed refresh log entry when the previous refresh failed.

Aha

> Would it look better if we just changed the text from "x seconds ago" to "just now"?

I think so, yes. It'd only change every minute, then.
